corehq/apps/fixtures/tests/test_views.py

```diff
@@ -282,3 +284,39 @@ def can_view_some_reports(domain):
                 "description": "A Table",
             }],
         )
+
+
+class UpdateTablesTests(TestCase):
+    def test_cannot_update_synced_fixture(self):
+        request = self._create_request('GET')
+        fixture = LookupTable.objects.create(domain='test-domain', tag='test-fixture', is_synced=True)
+
+        # ensure does not raise an exception
+        update_tables(request, 'test-domain', fixture.id)
+
+    def test_can_delete_synced_fixture(self):
+        request = self._create_request('DELETE')
+        fixture = LookupTable.objects.create(domain='test-domain', tag='test-fixture', is_synced=True)
+
+        # ensure does not raise an exception
+        update_tables(request, 'test-domain', fixture.id)
```

Comment:
Similar to the comment above, this test doesn't seem to verify that the lookup table was deleted. Should it be changed or renamed?

Reply:
Thanks for catching this. I think I had largely created a placeholder test and forgot about it. This has been updated to correctly verify that the table was deleted